### PR TITLE
Add maintainers.md file and update Readme for ARM commands

### PR DESCRIPTION
**Pull request by NimishaS**

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+This file explains the build and packaging commands of azure-chef-extension.
+
+##Contributors
```
Comment (adamedx) [inline MAINTAINERS.md]: This should be a maintainers section, and it should include me :smile:. Also, you may not want to add your entire team as maintainers if everyone hasn't spent a lot of time on it.

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+* [Siddheshwar More](https://github.com/siddheshwar-more)
+* [Nimisha Sharad](https://github.com/NimishaS)
+* [Vasundhara Jagdale](https://github.com/Vasu1105)
+* [Aliasgar Batterywala](https://github.com/Aliasgar16)
+
+##Build and Packaging
+You can use rake tasks to build and publish the new builds to Azure subscription.
+
+**Note:** The arguments have fix order and recommended to specify all for readability and avoiding confusion.
```
Comment (adamedx) [inline MAINTAINERS.md]: Should be "fixed" instead of "fix."

File: MAINTAINERS.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+#####Build
+    rake build[:target_type, :extension_version, :confirmation_required]
+
+:target_type = [windows/ubuntu/centos] default is windows
+
+:extension_version = Chef extension version, say 11.6 [pattern major.minor governed by Azure team]
```
Comment (adamedx) [inline MAINTAINERS.md]: We don't use version #'s like this anymore right? We should use current versioning scheme as an example to avoid confusion.

**Comment by adamedx** [comment]
So it seems like we moved build / deploy instructions from readme to maintainers.md -- not sure why. I think build and deploy should stay in readme. Maintainers is about who is responsible for doing work, reviewing requests, etc.  Here's an example of a good maintainers file: https://github.com/chef/chef/blob/master/MAINTAINERS.md.

**Comment by NimishaS** [comment]
ah ok :) I wasn't sure what exactly should come in maintainers.md. I moved the build/deploy instructions to maintainers.md because they are not for the end users but for the developers.

**Comment by adamedx** [comment]
:+1:

**Pull request merged**